en/console-and-shells/plugin-shell.rst

```diff
@@ -18,19 +18,19 @@ Via the `Load` task you are able to load plugins in your
 
 This will add the following to your **config/bootstrap.php**::
 
-    Plugin::load('MyPlugin', []);
+    Plugin::load('MyPlugin');
 
-By adding `-r` or `-b` to your command you can enable to `bootstrap` and
-`routes` value::
+Adding the `-b` or `-r` switch to the load task will enable loading of the plugin's
+`bootstrap` and `routes` values::
```

Comment:
If we do want to "keep" the highlighting, these will need to be double backticks.